### PR TITLE
[MIG] sale_order_line_packaging_qty: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

**Comment by simahawk** [comment]
to be taken into account here https://github.com/OCA/sale-workflow/pull/1276

**Comment by hparfr** [comment]
@phuctranerp what is the status of this PR ?

**Comment by hailangvn** [comment]
Thanks @simahawk , @hparfr . This PR has been updated. Please help to review. Thank you in advance.

File: sale_order_line_packaging_qty/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Sale Order Line Packaging Quantity",
+    "summary": "Define quantities according to product packaging on sale order lines",
+    "version": "14.0.0.1.0",
```
Comment (simahawk) [inline sale_order_line_packaging_qty/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

**Review by simahawk — CHANGES_REQUESTED** [review]
@TDu pls review

File: sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py [hunk 2]
```diff
@@ -67,3 +72,17 @@ def test_product_packaging_qty_wo_packaging(self):
         self.assertEqual(order_line.product_packaging_qty, 0.0)
         with self.assertRaises(UserError):
             order_line.write({"product_packaging_qty": 3.0})
+
+    def test_product_packaging_qty_from_external(self):
```
Comment (TDu) [inline sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py:76]: I am not sure to understand what more is tested with this function, than the beginning of the previous one ?

Comment (hailangvn) [reply sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py:76]: Thanks for the comment. The previous ones have `product_uom_qty` of 3, which is less than and not divisible by packaging qty of 5. This test is to increase coverage for the case that `product_uom_qty` of 15 is divisible by 5.

File: sale_order_line_packaging_qty/__manifest__.py (hunk 1 above)
Comment (hailangvn) [reply sale_order_line_packaging_qty/__manifest__.py]: Thanks for your suggestion. It is updated.

File: sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py (hunk 2 above)
Comment (TDu) [reply sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py:76]: Ok thanks for the explanation

**Review by TDu — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@phuctranerp there's a failing test on `sale_stock_picking_blocking` https://app.travis-ci.com/github/OCA/sale-workflow/jobs/524859223#L2116

**Comment by hailangvn** [comment]
> @simahawk your merge command was aborted due to failed check(s), which you can inspect on [this commit of 14.0-ocabot-merge-pr-1396-by-simahawk-bump-nobump](https://github.com/OCA/sale-workflow/commits/9de0dfe6606cd6cd36c5078a5cb385d8d55792a8).
> 
> After fixing the problem, you can re-issue a merge command. Please refrain from merging manually as it will most probably make the target branch red.

@simahawk , I checked the commit and saw that the travis using oca/OCB was green, while the travis using odoo/odoo was red. The issue caused by https://github.com/OCA/sale-workflow/blob/14.0/sale_stock_picking_blocking/tests/test_sale_stock_picking_blocking.py#L69. Further checking, that test uses newly created product. While this PR's test uses existing product. I am not sure what causes the conflict. Could you please advise how to troubleshoot? Thanks.

**Comment by simahawk** [comment]
you can try to install that module and see if you can reproduce it

**Comment by hailangvn** [comment]
Good day @simahawk, These are the things that I have tried. I still could not reproduce the error.
Good day @TDu, is there any suggestion how to find why travis is red with odoo/odoo repo? Any different between OCA/OCB and odoo/odoo repos?
* Pull the merged branch then install and test all modules in OCA/sale-workflow
* Pull the latest branch 14.0 from odoo/odoo
* export MAKEPOT=1
* Install Python 3.6 and test with it
* Load module in same sequence with failed test
* Test with commits 08583840604 (in which addons/mrp/models/stock_rule.py has same number of lines as in error log), e8ef346d574, 2da20136988, b3f67772dd0

**Comment by hailangvn** [comment]
Good day @simahawk, the issue can be fixed with https://github.com/OCA/sale-workflow/pull/1657. So this PR is now depending on that one. Thanks @nilshamerlinck for the help.

**Pull request merged**